AEPMessaging/Tests/TestHelpers/FunctionalTestBase.swift

```diff
@@ -12,6 +12,7 @@
 
 @testable import AEPCore
 import AEPServices
+import AEPTestUtils
```

Comment:
FunctionalTestBase + InstrumentedExtension are also available in TestUtils

Reply:
and a couple others too - thanks